### ISSUE TITLE
can not generate typedoc by using hierarchy

### ISSUE THREAD

**Issue by steven130169** [DiFuks/typedoc-theme-hierarchy#18]
My version is 

    "typedoc": "^0.24.7",
    "typedoc-theme-hierarchy": "^3.0.0",

This is my typedoc.json
```
{
  "entryPoints": [
    "src"
  ],
  "entryPointStrategy": "expand",
  "out": "./docs",
  "plugin": [
    "typedoc-theme-hierarchy"
  ],
  "theme": "hierarchy",
  "tsconfig": "./tsconfig.json",
  "name": "Gen2Gen-Backend",
  "exclude": [
    "**/*+(test|spec).ts"
  ],
  "categorizeByGroup": true
}
```

When I run command 
```
typedoc
```

the terminal show me error message
```
[info] Loaded plugin typedoc-theme-hierarchy
TypeDoc exiting with unexpected error:
TypeError: Cannot read properties of undefined (reading 'getValue')
    at defaultLayout (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/output/themes/default/layouts/default.js:6:130)
    at OverrideThemeContext.defaultLayout (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/output/themes/default/DefaultThemeRenderContext.js:33:22)
    at DefaultTheme.defaultLayoutTemplate (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/output/themes/default/DefaultTheme.js:37:53)
    at OverrideTheme.render (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/output/themes/default/DefaultTheme.js:168:37)
    at Renderer.renderDocument (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/output/renderer.js:183:40)
    at /Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/output/renderer.js:159:22
    at Array.forEach (<anonymous>)
    at Renderer.render (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/output/renderer.js:157:25)
    at async Application.generateDocs (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/application.js:300:9)
    at async run (/Users/steven-cheng/IdeaProjects/Gen2Gen-Backend/node_modules/.pnpm/typedoc@0.24.7_typescript@4.7.4/node_modules/typedoc/dist/lib/cli.js:110:13)
```

**Comment by Plopsi** [comment]
Seems Typedoc changed the signatures of classes/methods again. Here is a patch-diff for patch-package:

[typedoc-theme-hierarchy+3.2.0.patch](https://github.com/DiFuks/typedoc-theme-hierarchy/files/11674768/typedoc-theme-hierarchy%2B3.2.0.patch)

**Comment by DiFuks** [comment]
@Plopsi thnx! 

fixed in 4.0.0
